spec/expression.dd

```diff
@@ -2102,11 +2101,12 @@ void main()
     alias Tup = Tuple!(int, string);
     alias AA = long[char[]];
 
-    static if (is(Tup : TX!TL, alias TX, TL...))
+    static if (is(Tup : Template!Args, alias Template, Args...))
     {
-        writeln(is(TX!(int, long) == Tuple!(int, long)));  // true
-        writeln(typeid(TL[0]));  // int
-        writeln(typeid(TL[1]));  // immutable(char)[]
+        writeln(__traits(isSame, Template, Tuple)); // true
```

Comment:
Thanks, didn't know that. How does it know which commit to keep and which ones to squash?